source/azure/index.rst

```diff
@@ -0,0 +1,32 @@
+.. Copyright (C) 2018 Wazuh, Inc.
+
+.. _azure:
+
+Using Wazuh to Monitor Microsoft Azure
+======================================
+
+.. versionadded:: 3.7.0
+
+This section provides instructions for monitoring **Microsoft Azure** infrastructures. We will see how to monitor all activity that occurs in the infrastructure such as changes that occur in virtual machines, activation of alerts, health data information and other data from the control plane. We will show how to install Wazuh agents to monitor the virtual machines that make up the infrastructure, which will send events to the Wazuh manager for analysis in order to classify the event with a range of alerts that that can be easily viewed. We will also see how to monitor services of our infrastructure such as Azure Active Directory (AAD).
```

Comment:
health data information and other data from the control plane.

This could be "health data information as well as control data"